sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -347,6 +347,53 @@ export interface KeyVaultCertificateWithPolicy extends KeyVaultCertificate {
     readonly policy?: CertificatePolicy;
 }
 
+// @public
+export const enum KnownApiVersion72Preview {
```

Comment:
Why should this be exported?

Reply:
Similar to keyvault-keys, this value is not exported anymore. Removed it.